test/test_rack_handler.rb

```diff
@@ -226,4 +226,39 @@ def test_user_port_wins_over_config
       end
     end
   end
+
+  def test_file_log_requests_wins_over_deafult_config
+    file_log_requests_config = true
+
+    Dir.mktmpdir do |d|
+      Dir.chdir(d) do
```

Comment:
I'd rather just have a new, pre-baked config file on the filesystem, rather than create it dynamically. That way we can run the test in parallel.